app/assets/src/components/views/SampleViewV2/ReportTable.jsx

```diff
@@ -339,25 +342,38 @@ class ReportTable extends React.Component {
   };
 
   renderNtNrSelector = () => {
-    return this.renderNtNrStack({
-      cellData: ["NT", "NR"],
-      onClick: [
-        withAnalytics(
-          () => this.handleNtNrChange("nt"),
-          "ReportTable_count-type_clicked",
-          {
-            countType: "nt",
-          }
-        ),
-        withAnalytics(
-          () => this.handleNtNrChange("nr"),
-          "ReportTable_count-type_clicked",
-          {
-            countType: "nr",
-          }
-        ),
-      ],
-    });
+    return (
+      <BasicPopup
+        trigger={
+          <div>
```

Comment:
maybe you could move this trigger content into its own variable on the line above for readability, up to you